### DISCUSSION TITLE
Clarification on Mono's Support for ConcurrencyMode.Multiple in WCF Services

### DISCUSSION THREAD

**Discussion by yashnikam** [dotnet/runtime#110434] [Q&A]
Hello,

I am working on hosting a WCF service using Mono with ServiceHost and ServiceBehavior. I want to configure the service to handle multiple concurrent requests by setting ConcurrencyMode to Multiple.

Could you please confirm whether Mono fully supports ConcurrencyMode.Multiple and if there are any known limitations? Additionally, are there specific Mono versions or configurations required for this to work as expected?

Thank you for your assistance!

Best regards,
Yash N

**Comment by jkotas** [comment]
> I am working on hosting a WCF service using Mono 

If you are using Mono from https://github.com/mono/mono, you should ask this question in that repo or in new mono/mono home at GitLab.
